spec/factories/transaction.rb

```diff
@@ -8,6 +8,14 @@
 
     from_address_id { key.address.to_s }
 
+    block_hash { 'default_block_hash' }
+    block_height { '123' }
+    content do
+      Coinbase::Client::EthereumTransaction.new(
+        hash: 'transaction_hash'
+      )
+    end
```

Comment:
We should define a `trait` for transactions that are indexed.

E.g. we wouldn't expect these properties to be set on a `pending` transaction, but this definition makes them set on all factory transactions

Reply:
Similarly, we would not expect an `unsigned_payload`, `signed_payload`, etc... (probably) on an indexed transaction!

Reply:
added. thanks!